share/spice/forvo/forvo.js

```diff
@@ -159,3 +50,26 @@ Handlebars.registerHelper("forvo_sex", function(sex) {
     return "Female";
 });
 
+Handlebars.registerHelper('ifCond', function (v1, operator, v2, options) {
```

Comment:
You can find it here: https://github.com/duckduckgo/duckduckgo-template-helpers/blob/master/template_helpers.js#L11